lib/soapy_bing/accounts.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+module SoapyBing
+  class Accounts
+    attr_reader :oauth_credentials, :customer
+
+    def initialize(oauth: {}, customer: {})
+      @oauth_credentials = OauthCredentials.new(oauth)
+      @customer = Customer.new(customer)
+    end
+
+    def list
+      do_request(Soap::Request::GetAccountsInfoRequest).map do |account|
+        Account.new(
+          developer_token: customer.developer_token,
+          customer_id: customer.customer_id,
```

Comment:
I just noticed while implementing tests for bing ads gateway, that above two lines are not necessary, because of the say as `SoapyBing::Account` is implemented. It pull those values from `ENV` same as a `SoapyBing::Customer`.

From another side I don't really like idea of pulling things and prefer to push 'em. So I like this code and explicit assignment of values.

How do you feel? Would you like to do refactoring or remove those two lines? Both fine with me.